client/src/components/StyledText.tsx

```diff
@@ -97,6 +105,26 @@ function clearKeywordData() {
 function computeBasicKeywordData() {
     console.log("recomputed keyword data");
 
+    //add dummy names keywords
+    for(let i = 0; i < DUMMY_NAMES.length; i++) {
+        const name = DUMMY_NAMES[i];
+        KEYWORD_DATA_MAP["@"+(i+1)] = [
+            { style: "keyword-player-number", replacement: (i + 1).toString() },
+            { replacement: " " },
+            { style: "keyword-player-sender", replacement: name }
+        ];
+        KEYWORD_DATA_MAP["sender-"+name] = [
+            { style: "keyword-player-number", replacement: (i + 1).toString() },
+            { replacement: " " },
+            { style: "keyword-player-sender", replacement: name }
+        ];
+        KEYWORD_DATA_MAP[name] = [
+            { style: "keyword-player-number", replacement: (i + 1).toString() },
+            { replacement: " " },
+            { style: "keyword-player", replacement: name }
+        ];
+    }
+
```

Comment:
Lol, so I can send 1: Cotton Mather and it'll render? Kinda funny. Would be nice if there was a way to limit these dummy names to just the wiki tho...
I'm fine with merging like this though. Can be a TODO for later.
Add a TODO here for me

Reply:
no, make that an issue